tests/shell_quarter_BC/screen-output

```diff
@@ -0,0 +1,136 @@
+-----------------------------------------------------------------------------
```

Comment:
do you also want to add ``statistics`` file?

Reply:
Yes, I just gave the file the wrong name :-( Should be fixed now.